src/lib/storage/chunk.hpp

```diff
@@ -161,7 +161,7 @@ class Chunk : private Noncopyable {
    * However, `size() - invalid_row_count()` does not necessarily tell you how many rows are visible for
    * the current transaction.
    */
-  ChunkOffset invalid_row_count() const { return _invalid_row_count.load(); }
+  ChunkOffset invalid_row_count() const { return ChunkOffset{_invalid_row_count}; }
```

Comment:
Since `load` was added at three previous locations, shouldn't we be consistent here and also call `load`?

Reply:
I removed `load()` whenever possible as the direct access is guaranteed to do the same. However, some functions (e.g., `DTRACE` or `to_string`) do not know how to handle atomics. We thus explicitly obtain the underlying integral type.